protos/guard/guard.pb.go

```diff
@@ -676,17 +729,20 @@ func (*FileStoreMeta) XXX_MessageName() string {
 }
 
 type FileStoreStatus struct {
-	FileStoreMeta        *FileStoreMeta            `protobuf:"bytes,1,opt,name=file_store_meta,json=fileStoreMeta,proto3" json:"file_store_meta,omitempty" pg:"file_store_meta"`
-	State                FileStoreStatus_MetaState `protobuf:"varint,2,opt,name=state,proto3,enum=guard.FileStoreStatus_MetaState" json:"state,omitempty" pg:"state"`
-	Contracts            []*Contract               `protobuf:"bytes,3,rep,name=contracts,proto3" json:"contracts,omitempty" pg:"contracts"`
-	RenterSignature      []byte                    `protobuf:"bytes,4,opt,name=renter_signature,json=renterSignature,proto3" json:"renter_signature,omitempty" pg:"renter_signature"`
-	GuardReceiveTime     time.Time                 `protobuf:"bytes,5,opt,name=guard_receive_time,json=guardReceiveTime,proto3,stdtime" json:"guard_receive_time" pg:"guard_receive_time"`
-	ChangeLog            []*Log                    `protobuf:"bytes,6,rep,name=change_log,json=changeLog,proto3" json:"change_log,omitempty" pg:"change_log"`
-	CurrentTime          time.Time                 `protobuf:"bytes,7,opt,name=current_time,json=currentTime,proto3,stdtime" json:"current_time" pg:"current_time"`
-	GuardSignature       []byte                    `protobuf:"bytes,8,opt,name=guard_signature,json=guardSignature,proto3" json:"guard_signature,omitempty" pg:"guard_signature"`
-	XXX_NoUnkeyedLiteral struct{}                  `json:"-" pg:"-"`
-	XXX_unrecognized     []byte                    `json:"-" pg:"-"`
-	XXX_sizecache        int32                     `json:"-" pg:"-"`
+	FileStoreMeta        *FileStoreMeta              `protobuf:"bytes,1,opt,name=file_store_meta,json=fileStoreMeta,proto3" json:"file_store_meta,omitempty" pg:"file_store_meta"`
```

Comment:
make it none pointer, if it's not necessary to be a pointer. -- as instructed by gogo

Reply:
-- by set nullable to false